README.md

```diff
@@ -1,8 +1,44 @@
-# Instruqt packer image for K3s
+# Build the official Instruqt K3S images
 
-This repo contains the source files to build a K3s image for Instruqt.
+This repo builds offical Instruqt K3s images.
 
-What's includes:
+It runs at 3 am on the 15th of every month using GitHub actions.
```

Comment:
We need to change the readme as many things are no longer correct, but we can do this in another PR after it starts building.